.dockstore.yml

```diff
@@ -15,3 +15,6 @@ workflows:
   - subclass: WDL
     primaryDescriptorPath: /pipelines/mining/cytomining.wdl
     name: cytomining
+  - subclass: WDL
+    primaryDescriptorPath: /tests/cellpainting_workflow.wdl
+    name: cellpainting
```

Comment:
Looks good, unless we want to remove it entirely, but up to you

Reply:
I removed it completely :)